students/ak-foster/Lesson05/mailroom3.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+import os
+from datetime import datetime
+
+donate = {'Merv': [10, 500, 100], 'Linda': [300, 62, 7], 'Larry': [25, 50, 100], 'Sue': [20, 50, 1000]}
+
+
+def thankYou():
+    """Add new donations to record and display a thank you note when provided a name and donation amount."""
+
+    resp = input("Type 'list' to see donor list, otherwise, type a name or leave blank to quit:  ")
+    if resp == "":
```

Comment:
A cool thing about python is that None, False, empty strings, empty lists, etc all evaluate to False, so the preferred way to check for an empty string is `if not resp:`